pallets/pool-system/src/impls.rs

```diff
@@ -321,7 +309,7 @@ impl<T: Config> PoolMutate<T::AccountId, T::PoolId> for Pallet<T> {
 		Self::do_update_pool(&pool_id, &update.changes)?;
 
 		let num_tranches = pool.tranches.num_tranches().try_into().unwrap();
-		Ok(Some(T::WeightInfo::execute_update(num_tranches)).into())
+		Ok(num_tranches)
```

Comment:
We must have the same check as in the `update` logic.
I.e. check whether we are in submission period. Otherwise, an update would be dangerous for the rebalance and epoch closing.

Reply:
I thought here we get a scheduled updated, and check weather it's in the timeframe via:

```
		ensure!(
			Self::now() >= update.scheduled_time,
			Error::<T>::ScheduledTimeHasNotPassed
		);
```

But I will add the `InSubmissionPeriod` check here as well then!